src/js/records/requests/asset-update.record.js

```diff
@@ -40,6 +47,20 @@ export class AssetUpdateRequestRecord extends RequestRecord {
     this.attachedDetails = details
   }
 
+  /**
+   * Converts AssetRecord to AssetUpdateRequestRecord.
+   *
+   * @param {AssetRecord} assetRecord AssetRecord to be converted.
+   * @return {AssetUpdateRequestRecord} New AssetUpdateRequestRecord instance.
+   */
+  static fromAssetRecord (assetRecord) {
+    return new this({
+      details: {
+        assetUpdate: assetRecord,
```

Comment:
and again the same problem. It's better to have the method that accept `AssetRecord` and explicitly maps it's content to the assetUpdate field.

Reply:
and also see the bug here:

```
const assetRecord = new AssetRecord()
AssetUpdateRecord.fromAssetRecord(assetRecord) // <- passing the instance of `AssetRecord`, okay
```
then `fromAssetRecord` will call the constructor, that does:

```
this.asset = new AssetRecord(_get(
      this._record, 'details.assetUpdate' // <- but it's already `AssetRecord`!
))
```

Reply:
https://github.com/tokend/web-client/pull/37#discussion_r254729398